forge-gui/res/cardsfolder/upcoming/the_brothers_war.txt

```diff
@@ -0,0 +1,16 @@
+Name:The Brothers' War
+ManaCost:3 R
+Types:Enchantment Saga
+K:Saga:3:DBPowerStone,DBChoose,DBDamage
+SVar:DBPowerStone:DB$ Token | TokenAmount$ 2 | TokenTapped$ True | TokenScript$ c_a_powerstone | TokenOwner$ You | Description$  Create two tapped Powerstone tokens.
+SVar:DBChoose:DB$ ChoosePlayer | Defined$ You | Choices$ Player | AILogic$ Curse | RememberChosen$ True | SubAbility$ ChoosePTwo | SpellDescription$ Choose two target players. Until your next turn, each creature they control attacks the other chosen player each combat if able.
+SVar:ChoosePTwo:DB$ ChoosePlayer | Defined$ You | Choices$ NonChosenPlayer | AILogic$ Curse | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ MustAttackOne, MustAttackTwo | SubAbility$ DBCleanup | RememberObjects$ Remembered
+SVar:MustAttackOne:Mode$ MustAttack | EffectZone$ Command | ValidCreature$ Creature.RememberedPlayerCtrl | MustAttack$ ChosenPlayer
```

Comment:
I think you can make it work with single static:
`SVar:MustAttack:Mode$ MustAttack | EffectZone$ Command | ValidCreature$ Creature.RememberedPlayerCtrl+ActivePlayerCtrl | MustAttack$ Remembered.NonActive`